lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "packages": [
+    "packages/*"
+  ],
+  "version": "0.0.1"
```

Comment:
It really depends on what's going to be aside to `packages/css`.
What I understand is that there'll be a `packages/react` next to it, consuming `packages/css`. 
If so, my opinion is that it would be interesting to let the default lerna setting which is "fixed".

As a react consumer, I wouldn't want to have to check the internal version of the `css` package to know the used version as it can differ from the other one. (not the same lifecycle...) 

If most of the time, all packages change it would be better (IMHO) to make them evolve with the same version number even if sometimes it involves blank release on some packages.

Ps : that's what Babel does and it's working well 

(Also, as there is only one package for the moment we can leave it like this and decide when another package will be added)

Reply:
Completely agree @thollander.
Yes, after there will probably have `packages/react`, `packages/svelte`, `packages/wd` that will consume `packages/css` styles.
Except if there are some other suggestions inside our discussion about architecture in this repository.

Reply:
agree with @thollander.

In any case, it will be easier to switch from "fixed" mode to "independent" mode if you realize that the last one is more appropriate.